src/components/header/styles.css

```diff
@@ -0,0 +1,26 @@
+.main__header,
+.main__subheader {
+  margin: 0;
+  font-size: 2.625rem;
```

Comment:
Tablets (768px - 991px wide) could have a slightly smaller font, more similar in size to the mobile one.